deploy/manifests/prod/us-east-2/tenant/storetheindex/fdb-cluster/cluster.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps.foundationdb.org/v1beta2
+kind: FoundationDBCluster
+metadata:
+  name: fdb-prod-cluster
+spec:
+  version: 7.1.33
+  storageServersPerPod: 3
+  automationOptions:
+    replacements:
+      enabled: true
+  faultDomain:
+    key: kubernetes.io/hostname
+    valueFrom: spec.nodeName
+  databaseConfiguration:
+    storage_engine: ssd-rocksdb-v1
+  processCounts:
+    storage: 5
+    log: 2
+    stateless: 2
+  labels:
+    matchLabels:
+      foundationdb.org/fdb-cluster-name: fdb-prod-cluster
+    processClassLabels:
+      - foundationdb.org/fdb-process-class
+    processGroupIDLabels:
+      - foundationdb.org/fdb-process-group-id
+  minimumUptimeSecondsForBounce: 60
+  processes:
+    general:
+      customParameters:
+        - memory=20GiB
```

Comment:
Should this parameter be tuned to take up most or all of memory allocated to container?

Reply:
`storageServersPerPod` are set to 3 which leaves 20 GiB ish per process. Right?